L11-Streams/L11=Zad.3-MovieTheaterRoom/src/main/java/pl/babiarz/MovieTheaterRoom.java

```diff
@@ -0,0 +1,19 @@
+package pl.babiarz;
+
+public class MovieTheaterRoom {
+    private final String name;
```

Comment:
skoro typ sali to skończony zbiór (2D/3D/4D) to o wiele lepiej byłoby tu użyć enuma zamiast zwykłego stringa